### PR TITLE
feat: goldpinger: option to specify security context

### PR DESCRIPTION
**Pull request by sedflix**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped

**What this PR does / why we need it**:
For better security posture goldpinger deployment, we might need to specify security to run as non root etc.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
NA

**Special notes for your reviewer**:
@okgolove

### CODE REVIEW

File: charts/goldpinger/values.yaml [hunk 1]
```diff
@@ -93,6 +93,25 @@ podSecurityPolicy:
...
+## Set security context of the goldpinger container
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+containerSecurityContext:
+#  allowPrivilegeEscalation: false
+#  privileged: false
+#  readOnlyRootFilesystem: true
+
+## Set security context of the pod
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+podSecurityContext:
```
Comment (okgolove) [inline charts/goldpinger/values.yaml:105]: The same

File: charts/goldpinger/values.yaml [hunk 2]
```diff
@@ -93,6 +93,25 @@ podSecurityPolicy:
   enabled: false
   policyName: unrestricted-psp
 
+## Set security context of the goldpinger container
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
+containerSecurityContext:
```
[the remark this answers is not in the document]
Comment (sedflix) [reply charts/goldpinger/values.yaml:98]: @okgolove In my testing, either works well.

**Comment by okgolove** [comment]
@sedflix check CI result

**Comment by sedflix** [comment]
@okgolove fixed the issue

**Pull request merged**